proptest-regressions/layout/mod.txt

```diff
@@ -0,0 +1,8 @@
+# Seeds for failure cases proptest has generated in the past. It is
```

Comment:
Don't push this file; instead, convert it into a regular unit test. As far as I understand, these seeds change when changing the Op enum (happens frequently).

Reply:
removed in latest commit